VMS/Resources/scss/common/_variables.scss

```diff
@@ -7,12 +7,6 @@ $green: #64D72F;
 $green-bold: #306C14;
 $grey-bg: #E5E5E5;
 $grey-text: #515151;
-$greyicon: #B4B4B4;
-$grey: #F3F3F5;
-$blue-hv: #0B588c;
-$green-hv: #306C14;
-$grey-hv: #646464;
-
```

Comment:
these lines were deleted, please revert them

Reply:
I reverted them for @hoanguyennph